src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -185,12 +185,16 @@ public void readManifest() {
         eventTypes.put(ALERT_CEASED, EiffelAlertCeasedEvent.class);
         eventTypes.put(ALERT_RAISED, EiffelAlertRaisedEvent.class);
         eventTypes.put(ISSUE_DEFINED, EiffelIssueDefinedEvent.class);
-
         return eventTypes;
     }
 
     @Override
     public String generateMsg(String msgType, JsonObject bodyJson) {
+        return generateMsg(msgType, bodyJson, false);
+    }
+
+    @Override
+    public String generateMsg(String msgType, JsonObject bodyJson, Boolean lenientValidation) {
```

Comment:
Didn't we agree to not call this lenientValidation?

Reply:
from our latest discussion we are using two param names i.e 
Config param - lenientValidationEnabledToUsers
Rest Param - okToLeaveOutInvalidOptionalFields
Naming still we are using the lenient validation